orb/src/jobs/heroku-promote.yml

```diff
@@ -12,3 +22,5 @@ steps:
   - run:
       name: Deploy to production
       command: npx dotcom-tool-kit deploy:production
+  - change-api/generate:
```

Comment:
Question: it looks like the `change-api/generate` also uses `changeApiKey`, `user`, `environment`, `slackChannels` and `extraProperties` - is there a reason why they're not included here?

Reply:
The defaults seem fine to me so didn't think we needed to surface them as options – we can add them later if people need them